src/test/kotlin/com/hibob/academy/unitests/keren/ListManagerTest.kt

```diff
@@ -0,0 +1,140 @@
+package com.hibob.academy.unitests.keren
+
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.Test
+
+class ListManagerTest{
+    @Test
+    fun `Test adding a unique person`(){
+        assertEquals(true,ListManager().addPerson(Person("Ron Azar", 24)))
+    }
+
+    @Test
+    fun `Test adding a duplicate person and ensure it throws the expected exception`(){
+        assertThrows(IllegalArgumentException::class.java) {
+            val managers = ListManager()
+            managers.addPerson(Person("Ron Azar", 24))
+            managers.addPerson(Person("Ron Azar", 24))
+        }
+    }
+
+    @Test
+    fun `Test adding multiple people, checking that the list grows appropriately`(){
+        val managers = ListManager()
+        managers.addPerson(Person("Ron Azar", 24))
+        managers.addPerson(Person("Or Azar", 26))
+        assertEquals(2,managers.getManagersAmount())
+    }
+
+    //* Write tests for removePerson method:
+    @Test
+    fun `Test removing a person that exists in the list`(){
+        val managers = ListManager()
+        managers.addPerson(Person("Ron Azar", 24))
+        assertEquals(true,managers.removePerson(Person("Ron Azar",24)))
```

Comment:
Need to make sure it was actually removed

Reply:
Improved